chromium_src/chrome/browser/ui/views/chrome_browser_main_extra_parts_views_linux_x11.h

```diff
@@ -0,0 +1,5 @@
+#include "brave/browser/ui/views/brave_browser_main_extra_parts_views_linux.h"
+
+#define ChromeBrowserMainExtraPartsViewsLinux BraveBrowserMainExtraPartsViewsLinux  // NOLINT
```

Comment:
@simonhong what is adding this part fixing? Since `BraveBrowserMainExtraPartsViewsLinux` already existed, I'm confused why we're adding the override here. Surely it was already overriden to use it somewhere, or is this specifically to fix X11 vs another linux mode? Would be good to specify in commit message or in PR description if so.

Reply:
Yes, that makes ChromeBrowserMainExtraPartsViewsLinuxX11 also use 
BraveBrowserMainExtraPartsViewsLinux on linux with X11. PR description is updated. Thanks for review!